app/users/reducer.js

```diff
@@ -0,0 +1,92 @@
+import {
+  TOGGLE_LOGIN_MODE,
+  MANUAL_LOGIN_USER,
+  LOGIN_SUCCESS_USER,
+  LOGIN_ERROR_USER,
+  SIGNUP_USER,
+  SIGNUP_SUCCESS_USER,
+  SIGNUP_ERROR_USER,
+  LOGOUT_USER,
+  LOGOUT_SUCCESS_USER,
+  LOGOUT_ERROR_USER } from './actions';
```

Comment:
this was mainly to speed up my resolving merge conflict, happy to change it back.

Main reasons for keeping them within actions:
 - they are constants used to describe actions
 - generally all needed by actions
 - saving the 'type' label for adding type annotations :)